src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const rows = [...tbody.children];
+let sortedRows = [];
+
+document.querySelector('thead').addEventListener('click', e => {
+  sortTable(e.target.textContent);
+});
+
+function sortTable(textContent) {
+  switch (textContent) {
+    case 'Name':
+      sortedRows = rows.sort((a, b) =>
+        a.children[0].textContent.localeCompare(b.children[0].textContent));
+      break;
+    case 'Position':
+      sortedRows = rows.sort((a, b) =>
+        a.children[1].textContent.localeCompare(b.children[1].textContent));
+      break;
```

Comment:
avoid code duplicate. You can create function and use it (pass index you need)

Reply:
done